yarn.lock

```diff
@@ -4431,10 +4466,21 @@ run-async@^0.1.0:
   dependencies:
     once "^1.3.0"
 
+run-async@^2.2.0:
+  version "2.2.0"
+  resolved "https://registry.yarnpkg.com/run-async/-/run-async-2.2.0.tgz#8783abd83c7bb86f41ee0602fc82404b3bd6e8b9"
+  dependencies:
+    is-promise "^2.1.0"
+    pinkie-promise "^2.0.0"
+
 rx-lite@^3.1.2:
   version "3.1.2"
   resolved "https://registry.yarnpkg.com/rx-lite/-/rx-lite-3.1.2.tgz#19ce502ca572665f3b647b10939f97fd1615f102"
 
+rx@^4.1.0:
```

Comment:
rx is very large, I wonder how we feel about pulling it into Yarn. How much does this increase the installed size of Yarn? (you can run `scripts\build-dist.sh` on Linux/Mac or `scripts\build-dist.ps1` on Windows and compare the size of the `dist` directory before and after your change)